src/Telemetry/AzureDevOpsTimeline/Program.cs

```diff
@@ -52,18 +57,46 @@ private static void Main()
                                     parallelRequests = 5;
                                 }
 
-                                return new AzureDevOpsClient(
-                                    baseUrl: c["AzureDevOpsUrl"],
-                                    organization: c["AzureDevOpsOrganization"],
-                                    maxParallelRequests: parallelRequests,
-                                    accessToken: c["AzureDevOpsAccessToken"]
-                                );
+                                return new AzureDevOpsClientOptions
```

Comment:
In general, options should use "IOptions" and "Configure", rather than directly injecting an options object. Especially if secrets are involved.

Reply:
Replaced what we had with IOptions